src/en/discover/technology/index.html

```diff
@@ -58,15 +53,19 @@ <h2 class="h2 mb-12 lg:mb-16" id="object">Object storage</h2>
   <div class="grid lg:grid--cols-2 grid--gap-14 lg:grid--gap-28">
     <div>
       <p class="standout">
-        For businesses requiring seamless access to objects, Ceph provide comprehensive storage compatible with popular interfaces including
-        Amazon S3.
+      The Ceph Object Storage daemon (radosgw) is a FastCGI service that
+      provides a RESTful HTTP API to store objects and metadata. You can use
+      either the OpenStack Swift-compatible API or the Amazon S3-compatible
+      API, because the RADOS Gateway muses a unified namespace. This means that
+      you can write S3-compatible data with one application and read the same
+      data with another application that uses the Swift-compatible API.
```

Comment:
The Ceph RGW object storage service provides industry-leading S3 API compatibility with a robust set of security, tiering, and interoperability features.  Applications written to use S3 or Swift object storage can take advantage of Ceph's scalability and performance within a single datacenter, or federate multiple Ceph clusters across the globe to create a global storage namespace with an extensive set of replication, migration, and other data services.